confidant/authnz/__init__.py

```diff
@@ -238,6 +245,52 @@ def decorated(*args, **kwargs):
     return decorated
 
 
+def require_saml_role(credential):
```

Comment:
Would be nice for this to be `require_role`, rather than `require_saml_role`, so other modules can implement it, if necessary.

Reply:
Agreed